lib/active_merchant/billing/gateways/decidir.rb

```diff
@@ -119,6 +119,18 @@ def add_auth_purchase_params(post, money, credit_card, options)
         add_payment(post, credit_card, options)
       end
 
+      def add_payment_method_id(credit_card)
+        if options[:payment_method_id]
+          options[:payment_method_id].to_i
+        elsif CreditCard.brand?(credit_card.number) == 'cabal'
+          63
+        elsif CreditCard.brand?(credit_card.number) == 'naranja'
+          24
+        else
+          1
```

Comment:
do i correctly infer that 1 means "use the Luhn algorithm" and other numbers mean to use some other algorithm?

Reply:
Not to my knowledge, no. Although Naranja doesn't use Luhn, Cabal cards do follow Luhn. I honestly don't know what the significance of these numbers are; they are just what the rep at Decidir told me to use in an email.

Reply:
Herm, weird! Is the email in Zendesk? If so (or if in other easily accessible place) might be good to link to it from this PR, to provide context for posterity.

Reply:
Very excited to say that I was able to track this down in Decidir's documentation, which has been added to the commit message: https://decidirv2.api-docs.io/1.0/tablas-de-referencia-e-informacion-para-el-implementador/medios-de-pago-disponibles